tests/lightgbm/test_lightgbm_autolog.py

```diff
@@ -423,3 +429,42 @@ def test_lgb_autolog_continues_logging_even_if_signature_inference_fails(tmpdir)
     assert data is not None
     assert "run_id" in data
     assert "signature" not in data
+
+
+@pytest.mark.large
+def test_lgb_autolog_configuration_options(bst_params):
```

Comment:
We should also adopt @harupy 's suggestion from https://github.com/mlflow/mlflow/pull/3449/files#r494000413 in these lightgbm tests